oliviaruizknott/timer-rails
# Removes text when item is too short

This is a first step toward handling items that are ended so early that their elapsed time hasn’t yet passed the name and time written in the box. This is not the final solution, but at least it doesn’t look broken anymore.

## To Test
1. Start this meeting: https://salvadortimer.herokuapp.com/meetings/80
2. Click "Next Item" almost immediately after starting, before the elapsed time has reached the bottom of the word "First".
3. The text should be removed so that it doesn’t weirdly overflow. Nothing else happens at this time.
